### ISSUE TITLE
Import error when using latest version of pip dependency: ImportError: cannot import name 'GetAccessibilityRegion' from 'applitools.core.fluent.region'

### ISSUE THREAD

**Issue by tiborflach-finsync** [applitools/example-playwright-python-ufg#1]
Cloning the repo and using the older versions specified in "requirements.txt" does work.
However when upgrading to [latest pip version](https://pypi.org/project/eyes-playwright/5.25.10/) "eyes-playwright==5.25.10" I get the following import error:
```
E   ImportError: cannot import name 'GetAccessibilityRegion' from 'applitools.core.fluent.region'
```
I guess this is more like a request to please update the example test code to work with the latest dependency versions if applicable:
`eyes-playwright==5.25.10`
`pytest-playwright==0.5.0`

To recap: I am not sure if this is a bug, but I saw the latest pip version is quite ahead of the original example one and have not yet found related example code that shows how to use it.

Thank you!

**Comment by MJApplitools** [comment]
Hi @tiborflach-finsync, just pushed a fix for this issue. Thanks!
